main.go

```diff
@@ -78,7 +78,7 @@ func uploadApplications(configs Configs, service *androidpublisher.Service, appE
 func updateTracks(configs Configs, service *androidpublisher.Service, appEdit *androidpublisher.AppEdit, versionCodes []int64) error {
 	editsTracksService := androidpublisher.NewEditsTracksService(service)
 
-	newRelease, err := createTrackRelease(configs.WhatsnewsDir, versionCodes, configs.UserFraction)
+	newRelease, err := createTrackRelease(configs.WhatsnewsDir, versionCodes, configs.UserFraction, configs.uploadAsDraft)
```

Comment:
There is a typo which will prevent this code from compiling:
`main.go:81:106: configs.uploadAsDraft undefined (type Configs has no field or method uploadAsDraft`